modules/IsoMux/connection/connection.cpp

```diff
@@ -112,6 +112,10 @@ int check_interface(struct v2g_context* v2g_ctx) {
         v2g_ctx->if_name = choose_first_ipv6_interface();
     }
 
+    if (v2g_ctx->if_name == nullptr) {
```

Comment:
In line 125 there is already a check if `if_name == nulltpr`. So the segfault is happening in line 119 (`mreq.ipv6mr_interface = if_nametoindex(v2g_ctx->if_name);`) ?

Reply:
exactly